proto/kvrpcpb.proto

```diff
@@ -593,6 +593,45 @@ message UnsafeDestroyRangeResponse {
     string error = 2;
 }
 
+message RegisterLockObserverRequest {
+    Context context = 1;
+    uint64 max_ts = 2;
+}
+
+message RegisterLockObserverResponse {
+    string error = 1;
+}
+
+message CheckLockObserverRequest {
+    Context context = 1;
+    uint64 max_ts = 2;
+}
+
+message CheckLockObserverResponse {
+    string error = 1;
+    bool is_clean = 2;
+    repeated LockInfo locks = 3;
+}
+
+message RemoveLockObserverRequest {
+    Context context = 1;
+    uint64 max_ts = 2;
+}
+
+message RemoveLockObserverResponse {
+    string error = 1;
+}
+
+message PhysicalScanLockRequest {
+    Context context = 1;
```

Comment:
Do we really need it?

Reply:
No. But since I'm afraid that in the future there might be more fields  in context that might be useful for these requests, I kept the context for all of them. I also kept context for `UnsafeDestroyRangeRequest`.